api/v1alpha4/azuremachine_conversion.go

```diff
@@ -17,20 +17,41 @@ limitations under the License.
 package v1alpha4
 
 import (
+	apimachineryconversion "k8s.io/apimachinery/pkg/conversion"
```

Comment:
let's keep the import naming consistent with other packages

```suggestion
	apiconversion "k8s.io/apimachinery/pkg/conversion"
```